netcdf_to_gltf_converter/config.py

```diff
@@ -99,6 +99,36 @@ class ModelType(StrEnum):
     XBEACH = "XBEACH"
     """Output from an XBEACH model (regular grid)."""
 
+class ShiftType(StrEnum):
+    """The method to shift the coordinates."""
+    
+    MIN = "min"
+    """The smalles x- and -y coordinate become the origin (0,0,z)."""
+    
+class CrsTransformation(BaseModel):
+    """The configuration settings for transforming the coordinates."""
+
+    source_epsg: int
+    """int: EPSG code of the source coordinate system."""
+
+    target_epsg: int
+    """int: EPSG code of the target coordinate system."""
+
+class CrsShifting(BaseModel):
```

Comment:
Does Crs stand for something that is well know in the context of this software?

Maybe the name 'CoordinatesShiftingSettings' might be a more descriptive name?

Reply:
As explained: CRS is a commonly used abbreviation for coordinate system